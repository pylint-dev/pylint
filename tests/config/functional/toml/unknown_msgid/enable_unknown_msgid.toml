# Check the behavior for unkonwn symbol/msgid
# (Originally) reported in https://github.com/PyCQA/pylint/pull/6293

[tool.pylint."messages control"]
disable = "logging-not-layzy,logging-format-interpolation"
enable = "locally-disabled,C00000"
