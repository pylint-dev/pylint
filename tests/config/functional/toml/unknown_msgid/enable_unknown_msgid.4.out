************* Module {abspath}
{relpath}:1:0: W0012: Unknown option value for '--disable', expected a valid pylint message and got 'logging-not-layzy' (unknown-option-value)
{relpath}:1:0: W0012: Unknown option value for '--enable', expected a valid pylint message and got 'C00000' (unknown-option-value)
